lib/operations/drop.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const applyWriteConcern = require('../utils').applyWriteConcern;
+const Aspect = require('./operation').Aspect;
+const CommandOperation = require('./command');
+const defineAspects = require('./operation').defineAspects;
+const handleCallback = require('../utils').handleCallback;
+
+class DropOperation extends CommandOperation {
```

Comment:
I thought it made sense because they are both doing the same options handling in the constructor. You're correct that they both need the write concern applied, which is handled by `CommandOperation.execute`. I'll remove `applyWriteConcern` here since it's not necessary to do it twice.